sequencing/ATACSeq/jobSubmission/1_batchRunPreAnalysis.sh

```diff
@@ -0,0 +1,159 @@
+#!/bin/sh
```

Comment:
Could you change this to use bash instead of sh?
Just for consistency across the repository